Source/Analytics/Web.Frontend/src/components/Map.js

```diff
@@ -90,13 +90,15 @@ class MapWidget extends Component {
             );
         }
         return (
-            <Map center={position} zoom={6}>
+            <Map center={position} zoom={5} style={{height: 210, width: 210}}>
+                {/* TODO: Auto zoom and center based on set of markers */}
```

Comment:
Can this be moved to an issue instead of a comment?

Reply:
Yep!